### PR TITLE
chore(core): unified query logging

### PR DESCRIPTION
**Pull request by glasstiger**
introduced a query logger which is used wherever a query is parsed, or taken from the cache for execution.
also cleaned up ILP commit related logging a bit.

required by https://github.com/questdb/questdb-enterprise/pull/206

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 44 / 44 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/DefaultQueryLogger.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/QueryLogger.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cairo/CairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|5|5|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 44 / 44 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/DefaultQueryLogger.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/QueryLogger.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cairo/CairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|5|5|100.00%|

File: core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java [hunk 1]
```diff
@@ -737,7 +737,6 @@ private TableUpdateDetails getTableUpdateDetailsFromSharedArea(
                         }
                     }
                     tableToken = engine.createTable(securityContext, ddlMem, path, true, tsa, false);
-                    LOG.info().$("created table [tableName=").$(tableNameUtf16).I$();
```
Comment (jerrinot) [inline core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java:740]: is this intentionally removed?

Comment (glasstiger) [reply core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java:740]: yes, it is redundant.

for OSS we already log this:
```
2023-10-25T22:12:01.986061Z I i.q.c.p.WriterPool created [table=`test~8`, thread=27]
```

for Ent we will have the same as above + EntDddlListener adds its own line which contains the username.

**Review by jerrinot — CHANGES_REQUESTED** [review]
looks good. please add `TextQueryProcessor` as discussed and it's good to merge.

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 44 / 44 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/DefaultQueryLogger.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/QueryLogger.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cairo/CairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|5|5|100.00%|

**Comment by glasstiger** [comment]
> looks good. please add `TextQueryProcessor` as discussed and it's good to merge.

pushed the change

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 49 / 49 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/JsonQueryProcessor.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/DefaultQueryLogger.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlExecutionContextImpl.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/QueryLogger.java|9|9|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompilerImpl.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/cairo/CairoConfiguration.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/pgwire/PGConnectionContext.java|5|5|100.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/TextQueryProcessor.java|5|5|100.00%|

**Review by jerrinot — APPROVED** [review]

**Pull request merged**